app/services/fee_scheme.rb

```diff
@@ -0,0 +1,27 @@
+class FeeScheme
+  def initialize(type, date)
+    @type = type
+    @date = date
+  end
+
+
+  def fee_scheme
+    @fee_scheme ||= client.fee_schemes(type: @type, case_date: @date)
+  end
+
+  def calculate
+    fee_scheme.calculate(
+      scenario: 5,
+      advocate_type: 'JRALONE',
+      offence_class: 'E',
+      fee_type_code: 'AGFS_APPEAL_CON',
+      day: 1,
+      number_of_defendants: 1,
+      number_of_cases: 1
```

Comment:
Is this test data? Might be better defining it in `spec/services/fee_scheme_spec.rb` instead of the service class?

Reply:
This is just checking to see the call is being made to the API from the backend

Reply:
I think as long as it's temporary it's probably ok. Let's leave it for now, I don't want to block the progress you're making 🙂